src/main/kotlin/com/example/realworldkotlinspringbootjdbc/infra/CommentRepositoryImpl.kt

```diff
@@ -1,7 +1,91 @@
 package com.example.realworldkotlinspringbootjdbc.infra
 
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import com.example.realworldkotlinspringbootjdbc.domain.ArticleId
+import com.example.realworldkotlinspringbootjdbc.domain.Comment
 import com.example.realworldkotlinspringbootjdbc.domain.CommentRepository
+import com.example.realworldkotlinspringbootjdbc.domain.article.Slug
+import com.example.realworldkotlinspringbootjdbc.domain.comment.Body
+import com.example.realworldkotlinspringbootjdbc.domain.comment.CommentId
+import com.example.realworldkotlinspringbootjdbc.domain.user.UserId
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
 import org.springframework.stereotype.Repository
+import java.text.SimpleDateFormat
 
 @Repository
-class CommentRepositoryImpl : CommentRepository
+class CommentRepositoryImpl(val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) : CommentRepository {
+    override fun list(slug: Slug): Either<CommentRepository.ListError, List<Comment>> {
+        /**
+         * article を取得
+         */
+        val selectArticleSql = """
+            SELECT
+                id
+            FROM
+                articles
+            WHERE
+                slug = :slug
+        """.trimIndent()
```

Comment:
[nits&imo]

1. [nits] セミコロンお願いします 🙇 （というかこれ通ったのですね）
2. [imo] `AS` をつけるのはどうでしょう（後述のcommentのSQLではつけてるため、1つのメソッドの中でで統一されてたらどちらでも良い）
3. [imo] WHEREで `articles.slug` とするのはどうでしょう？（後述のcommentのSQLではつけてるため、1つのメソッドの中でで統一されてたらどちらでも良い）

```suggestion
        val selectArticleSql = """
            SELECT
                id AS id
            FROM
                articles
            WHERE
                articles.slug = :slug
            ;
        """.trimIndent()
```

Reply:
https://github.com/sunakan/realworld-kotlin-springboot-jdbc/pull/107/commits/e8c5548048cf05c479b6517e3d690273a79ac48f